crates/web/src/components/text_field.rs

```diff
@@ -82,7 +85,7 @@ pub fn TextField(
     view! {
             <div>
             <label class="block mb-2 text-sm font-medium text-purple-500" for=id.clone()>{label}</label>
-            <input type=format!("{}", r#type) name=name value=value id=id placeholder=placeholder class=class_names  disabled=disabled />
+            <input type=format!("{}", r#type) name=name value=value id=id placeholder=placeholder class=class_names  disabled=disabled on:change=move |ev| {console_log(event_target_value(&ev).as_str())}  />
```

Comment:
I think you want to use `on:input` here, based on: https://book.leptos.dev/view/05_forms.html#controlled-inputs

Reply:
Yeah! I am searching how to share the value when the component is used

Reply:
I think your best take is to use https://docs.rs/leptos/latest/leptos/callback/index.html.

You could have a prop `on_input` which is a instance of `Callback<String>` and use this `on_input.call(ev.target.value)` there I think.